cmd/context/selector.go

```diff
@@ -92,17 +88,32 @@ func getK8sClusters(k8sClusters []string) []utils.SelectorItem {
 	orderedK8sClusters := make([]utils.SelectorItem, 0)
 	for _, k8sCluster := range k8sClusters {
 		orderedK8sClusters = append(orderedK8sClusters, utils.SelectorItem{
-			Name:      k8sCluster,
-			Label:     k8sCluster,
-			Enable:    true,
-			IsOkteto:  false,
-			Namespace: getKubernetesContextNamespace(k8sCluster),
-			Builder:   "docker",
-			Registry:  "-",
+			Name:   k8sCluster,
+			Label:  k8sCluster,
+			Enable: true,
 		})
 	}
 	sort.Slice(orderedK8sClusters, func(i, j int) bool {
 		return strings.Compare(orderedK8sClusters[i].Name, orderedK8sClusters[j].Name) < 0
 	})
 	return orderedK8sClusters
 }
+
+func getInitialPosition(options []utils.SelectorItem) int {
+	currentContext := okteto.ContextStore().CurrentContext
+	for indx, item := range options {
+		if item.Enable && item.Name == currentContext {
+			return indx
+		}
+	}
+	return -1
+}
+
+func selectedIsOkteto(selected string) bool {
```

Comment:
Can it be named only `isOkteto`? `selected` gives it a context where the function can be used, but at the end, this is only checking if a context belongs to Okteto or not and it can be used in other places

Reply:
i've been thinking and moved this check to the okteto pkg, i think it makes more sense